marklogic-data-hub-central/ui/src/components/modeling/graph-view/properties-tab/properties-tab.tsx

```diff
@@ -1,22 +1,61 @@
-import React from "react";
+import React, {useState} from "react";
 import PropertyTable from "../../property-table/property-table";
+import styles from "./properties-tab.module.scss";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faLayerGroup, faKey} from "@fortawesome/free-solid-svg-icons";
+import arrayIcon from "../../../../assets/icon_array.png";
 
 interface Props {
   entityTypeData: any;
   canWriteEntityModel: any;
   canReadEntityModel: any;
 }
 
+
 const PropertiesTab: React.FC<Props> = (props) => {
+  const [showLegend, setShowLegend] = useState(false);
+
+  const toggleLegend = () => {
+    if (showLegend) {
+      setShowLegend(false);
+    } else {
+      setShowLegend(true);
+    }
+  };
+
+  const LegendShowHide =
+    <div>
+      {showLegend ?
+        <div className={styles.legend}>
+          <div data-testid="foreignKeyIconLegend" className={styles.foreignKeyLegendText}>
+            <FontAwesomeIcon className={styles.foreignKeyIcon} icon={faKey}/> Foreign
+                                  Key Relationship
```

Comment:
Can we clean up the label content so that it is always together (same line either with the icon or not)?